src/stylesheets/components/global/column.scss

```diff
@@ -0,0 +1,78 @@
+.column__container {
+    display: flex;
+    flex-direction: column;
+}
```

Comment:
So here you would do

```
.column-container {
    display: flex
    flex-direction: column;
    
    &__heading {
     ....
    }
    
    &__sorting {
    ...
    }
}
```
The & symbol is equal to the top-level class name, and this helps you structure the scss files in the same way the class names appear in the component. Handy for error-checking etc.